h/models/annotation_slim.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+
+import sqlalchemy as sa
+
+from h.db import Base, types
+
+
+class AnnotationSlim(Base):
+    """
+    AnnotationSlim represents fundamental properties of annotations.
```

Comment:
I think `AnnotationMetadata` is probably the best name.

I can see what you're getting at with "fundamental properties" but I think I'd consider some of the properties that _aren't_ in this table to be fundamental properties of an annotation (e.g. the actual annotation text itself) whereas I think I'd consider some of the properties that _are_ in this table to be _not_ fundamental, e.g. `moderated` etc.

I think "metadata" is probably the word that most closely captures what's in this table.